image/lxc.go

```diff
@@ -96,7 +96,7 @@ func (l *LXCImage) createMetadata() error {
 	}
 
 	err = l.writeMetadata(filepath.Join(metaDir, "expiry"),
-		string(shared.GetExpiryDate(time.Now(), l.definition.Expiry).Unix()))
+		fmt.Sprint(shared.GetExpiryDate(time.Now(), l.definition.Expiry).Unix()))
```

Comment:
Hm, wouldn't make an explicit `Sprintf("%d)` make more sense since it avoids Go reflection? Maybe I'm overthinking this.